docs/modules/reference/pages/service-assurance/monitors/HttpMonitor.adoc

```diff
@@ -110,54 +110,61 @@ This monitor implements the <<reference:service-assurance/introduction.adoc#ref-
-  <parameter key="response" value="200-202,299"/>
-  <parameter key="response-text" value="~.*[Cc]onsulting.*"/>
+  <parameter key="retry" value="1"/> <1>
+  <parameter key="timeout" value="3000"/> <2>
+  <parameter key="port" value="80"/> <3>
+  <parameter key="host-name" value="opennms.com"/> <5>
+  <parameter key="url" value="/solutions"/> <4>
+  <parameter key="response" value="200-202,299"/> <6>
+  <parameter key="response-text" value="~.*[Cc]onsulting.*"/> <7>
 </service>
 
 <!-- Test for instance of OpenNMS running -->
 <service name="OpenNMS-129" interval="300000" user-defined="false" status="on">
-  <parameter key="retry" value="1"/>
-  <parameter key="timeout" value="3000"/>
-  <parameter key="port" value="8080"/>
-  <parameter key="url" value="/opennms/event/list"/>
-  <parameter key="basic-authentication" value="admin:admin"/>
-  <parameter key="response" value="200"/>
+  <parameter key="retry" value="1"/> <1>
+  <parameter key="timeout" value="3000"/> <2>
+  <parameter key="port" value="8080"/> <3>
+  <parameter key="url" value="/opennms/event/list"/> <4>
+  <parameter key="basic-authentication" value="admin:admin"/> <8>
+  <parameter key="response" value="200"/> <9>
 </service>
 
 <!-- Test for instance of OpenNMS with parameter substitution in basic-authentication parameter -->
 <service name="OpenNMS-22" interval="300000" user-defined="false" status="on">
-  <parameter key="retry" value="1"/>
-  <parameter key="timeout" value="3000"/>
-  <parameter key="port" value="8080"/>
-  <parameter key="url" value="/opennms/event/list"/>
-  <parameter key="basic-authentication" value="{username}:{password}"/>
-  <parameter key="response" value="200"/>
+  <parameter key="retry" value="1"/> <1>
+  <parameter key="timeout" value="3000"/> <2>
+  <parameter key="port" value="8080"/> <3>
+  <parameter key="url" value="/opennms/event/list"/> <4>
+  <parameter key="basic-authentication" value="{username}:{password}"/> <8>
+  <parameter key="response" value="200"/> <9>
 </service>
-<monitor service="HTTP" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor" /> <1>
-<monitor service="OpenNMSdotCom" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor" /> <1>
-<monitor service="OpenNMS-129" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor" /> <1>
-<monitor service="OpenNMS-22" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor" /> <1>
+<monitor service="HTTP" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor" /> <10>
+<monitor service="OpenNMSdotCom" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor" /> <10>
+<monitor service="OpenNMS-129" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor" /> <10>
+<monitor service="OpenNMS-22" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor" /> <10>
 ----
-<1> Required *monitor* section for each service.
+<1> Number of attempts to test a service’s status
+<2> Timeout for the isReachable method, in milliseconds.
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.